auto_nag/bugbug_utils.py

```diff
@@ -68,15 +60,6 @@ def bughandler(self, bug, data):
             return
         data.append(bugid)
 
-    def remove_using_history(self, bugs):
-        return bugs
-
-    def failure_callback(self, bugid):
-        self.to_cache.remove(int(bugid))
-
-    def terminate(self):
-        self.add_to_cache(self.to_cache)
-
     def get_bugs(self, model, date="today", bug_ids=[], retry_count=100, retry_sleep=1):
```

Comment:
I think this function is no longer used by anyone, we should be able to remove it

Reply:
Maybe the whole class is no longer necessary, you could move the needed things (probably few) into the only script that is still extending the class (regressor)